lib/tui/init/dune

```
@@ -0,0 +1,7 @@
+(library
+ (name init)
+ (libraries
+  ;; Internal dependencies
+  terminal_size
```

`terminal_size` is an external dependency:
- https://github.com/cryptosense/terminal_size
```suggestion
  terminal_size
  ;; Internal dependencies
```